README.md

```diff
@@ -58,6 +58,14 @@ Office hours TBD.
 *   No exams
 
 Problem sets consist of code. MIT students enrolled in the course must submit homeworks via Canvas. If you are not a student then we encourage you to join the Discord and find a cross-grading partner.
+
+### Quickstart
```

Comment:
Maybe "Quick start using Docker"

Reply:
> Thanks! In my experience just installing and running Docker is rather tricky.

I did consider that this might cause more issues than it fixes. Windows Home edition doesn't allow virtualization, so this might not be usable by a lot of students. I'm fine with just letting this sit, or if you would rather close the request I'm cool with that too.